### PR TITLE
Update `add-new-source.md`

### PR DESCRIPTION
**Pull request by joeflack4**
## Overview
New section for components/, with information about:
- `property-map.sssom.tsv`'s
- SPARQL queries
- `properties.txt`

### Documentation
Was the documentation added/updated under `docs/`?
- [x] Yes
- [ ] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [ ] Yes
- [x] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [x] Yes

---

CC @souzadevinicius: Take a read over this to familiarize yourself with the process of adding a new source to Mondo.

### CODE REVIEW

File: docs/developer/add-new-source.md [hunk 1]
```diff
@@ -50,7 +50,24 @@ Run `sh run.sh make ../../docs/metrics/*.md` from `src/ontology`.
 ### 3.5. `docs/metrics.md`
 Add a link to your new `.md` file created in the last step.
 
-## 4. Everything else
+## 4. The `components/` goal
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:53]: ## New `components/` sectoin

File: docs/developer/add-new-source.md [hunk 2]
```diff
@@ -50,7 +50,24 @@ Run `sh run.sh make ../../docs/metrics/*.md` from `src/ontology`.
...
+When `mondo-ingest-odk.yaml` is edited and `update_repo` is run, a new `component-download-SOURCE.owl` goal is added to 
+`src/ontology/Makefile`. This essentially downloads a copy of the source. But, for it to be used in `mondo-ingest`, it 
+needs to be modified. This is done by creating a `$(COMPONENTSDIR)/omim.owl` goal in `mondo-ingest.Makefile` which runs 
+a composite `robot` command. Here are some sections you'll likely want to add that command:  
+- `rename --mappings config/SOURCE-property-map.sssom.tsv`: You'll notice that there are two other lines on most goals
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:58]: ## `property-map.sssom.tsv` subsection
Trish and I came to a determination in #482, and that is being codified here a bit.

File: docs/developer/add-new-source.md [hunk 3]
```diff
@@ -50,7 +50,24 @@ Run `sh run.sh make ../../docs/metrics/*.md` from `src/ontology`.
...
+`src/ontology/Makefile`. This essentially downloads a copy of the source. But, for it to be used in `mondo-ingest`, it 
+needs to be modified. This is done by creating a `$(COMPONENTSDIR)/omim.owl` goal in `mondo-ingest.Makefile` which runs 
+a composite `robot` command. Here are some sections you'll likely want to add that command:  
+- `rename --mappings config/SOURCE-property-map.sssom.tsv`: You'll notice that there are two other lines on most goals 
+for `property-map.sssom.tsv` and `property-map-2.sssom.tsv`. These handle some common renamings. However, 
+if your source has any properties that need to be renamed which are specific to that source, then you should create a 
+new file for that. While doing this, it might be useful to check any other `SOURCE-property-map.sssom.tsv` files to see 
+if they share any of the same property renamings. If so, it might be a good idea to move them to 
+`property-map.sssom.tsv` instead.
+- `query --update ../sparql/SPARQL.ru`: There may be more complex modifications that need to be made to the ontology. If
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:64]: ## SPARQL update subsection

File: docs/developer/add-new-source.md [hunk 4]
```diff
@@ -50,7 +50,24 @@ Run `sh run.sh make ../../docs/metrics/*.md` from `src/ontology`.
...
+ so, this can be done by 1 or more SPARQL queries.
+- `remove -T config/properties.txt --select complement --select properties --trim true`: `properties.txt` contains a
```
Comment (joeflack4) [inline docs/developer/add-new-source.md:66]: ## `properties.txt` subsection

**Review by matentzn — APPROVED** [review]
LGTM

**Pull request merged**